_docs/tutorials/getting-started-react.md

```diff
@@ -16,7 +16,7 @@ First of all you need a sensenet instance. You can create your own from GitHub o
 
 ## Use our starter
 
-To make things easier, we created a boilerplate app. It contains an example Hello World React SPA with sensenet repository login, written in Typescript. It is a template repository that you can find here [sn-react-typescript-boilerplate](https://github.com/SenseNet/sn-react-typescript-boilerplate). _If you don't know how to use a template repository check this [article](https://help.github.com/en/articles/creating-a-repository-from-a-template)_.
+To make things easier, we created a boilerplate app. It contains an example Hello World React SPA with sensenet repository login, written in Typescript. It is a template repository that you can find here [sn-react-typescript-boilerplate](https://github.com/SenseNet/sn-client/tree/master/examples/sn-react-typescript-boilerplate). _If you don't know how to use a template repository check this [article](https://help.github.com/en/articles/creating-a-repository-from-a-template)_.
```

Comment:
```suggestion
To make things easier, we created a boilerplate app. It contains an example Hello World React SPA with sensenet repository login, written in Typescript. You can find here [sn-react-typescript-boilerplate](https://github.com/SenseNet/sn-client/tree/master/examples/sn-react-typescript-boilerplate).
```

Reply:
fixed on 0682e100dc19cdb1fdbb633e6eca45d8e76c16e5

thanks

Reply:
Can you also remove the last sentence from each paragraph? Thank you :)

Reply:
Sorry I didn't see it. Fixed on cffa5c0

Thanks